pftools/python/parflow/tools/builders.py

```diff
@@ -801,6 +810,32 @@ def db_prefix(self):
         return 'subsurface_'
 
 
+class WellPropertiesBuilder(TableToProperties):
```

Comment:
This class defines the builder we use. It inherits from the same parent as the other builders